frontend/src/ContentArea/Root.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default class Root extends React.Component {
+  render() {
+    const root = JSON.parse(localStorage.getItem('tree'));
+
+    let keys = Object.keys(root).filter(k => k !== 'titleDisplay');
+    keys = keys.map(k => {
+      const obj = {
+        path: k,
+        name: root[k].titleDisplay,
+      };
```

Comment:
Here I'd prefer `reduce` instead of `filter` + `map`. Also it could just return JSX instead of calling `map` again below.